app/utils/agendaitem-utils.js

```diff
@@ -195,6 +195,7 @@ export const reorderAgendaitemsOnAgenda = async(agenda, store, decisionReportGen
   const actualAnnouncements = [];
   const sortedAgendaitems = agendaitems.slice().sort((a1, a2) => a1.number - a2.number)
   for (const agendaitem of sortedAgendaitems) {
+    await agendaitem.belongsTo('type').reload(); // type could have changed concurrently
```

Comment:
scenario where user 1 is removing/sending back an agendaitem after user 2 changed a type of an agendaitem.
There are more locations where we are just relying on type being the same.
Without this:
user 1 and 2 start from agenda view
user 2 changes agendaitem 2 to announcement
user 1 removes/sends an agendaitem back.
During the process for user 1, we check every single agendaitem and assign a new number based in the index.
But since the former nota is still in the nota list, the numbering of the notas is wrong